test/parallel/test-global-customevent.js

```diff
@@ -0,0 +1,16 @@
+// Flags: --experimental-global-customevent --expose-internals
+'use strict';
+
+const common = require('../common');
+const { strictEqual, ok } = require('node:assert');
+const { Worker, isMainThread } = require('worker_threads');
+const { CustomEvent } = require('internal/event_target');
+
+ok(CustomEvent);
+strictEqual(globalThis.CustomEvent, CustomEvent);
+
+if (isMainThread) {
+  const w = new Worker(__filename);
+  w.on('message', common.mustNotCall());
+  w.on('error', common.mustNotCall());
+}
```

Comment:
I don't think this is necessary, we run all the parallel tests in a Worker in the `test-worker` job anyway.
```suggestion
```